src/applications/lgy/coe/form/config/chapters/service/status.js

```diff
@@ -10,7 +10,7 @@ schema.properties.identity.enumNames = [
   </>,
   'I’m an active-duty service member',
   <>
-    I'm a <strong>current</strong> member of the National Guard{' '}
+    I'm a <strong>current</strong> member of the National Guard or Reserves{' '}
```

Comment:
Weird that eslint didn't catch this apostrophe

Reply:
Nice catch! I went ahead and fixed all of the single quotes in the file